codegen/MonocleCodegen.hs

```diff
@@ -168,9 +172,13 @@ protoToServant pb =
 
     api =
       [ "type MonocleAPI =",
-        Text.intercalate "\n :<|>" $ map mkApi methods
+        Text.intercalate "\n :<|>" $ (map mkApi methods ) <> otherEndpoints
       ]
       where
+        otherEndpoints = [
+          "\"auth\" :> \"login\" :> Get '[JSON] NoContent",
+          "\"auth\" :> \"cb\" :> QueryParam \"error\" Text :> QueryParam \"code\" Text :> QueryParam \"state\" Text :> Get '[HTML] LoginInUser"
+          ]
```

Comment:
The codegen shouldn't have to deal with non-protobuf endpoint. Instead we could add the extra endpoints in the Monocle.Main module.